CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 - Fix broken codesign option for bundle dependency [#1104](https://github.com/yonaskolb/XcodeGen/pull/1104) @kateinoigakukun
 - Ensure fileTypes are mapped to JSON value [#1112](https://github.com/yonaskolb/XcodeGen/pull/1112) @namolnad
 - Fix platform filter for package dependecies [#1123](https://github.com/yonaskolb/XcodeGen/pull/1123) @raptorxcz
+- Fix Xcode 13 build [#1127](https://github.com/yonaskolb/XcodeGen/issues/1127) @raptorxcz
```

Comment:
Please move it to Next Version

Reply:
🤔 version 2.25.0 was not released.

Reply:
but maybe would be better to review of this PR https://github.com/yonaskolb/XcodeGen/pull/1130 instead of mine.

Reply:
https://github.com/yonaskolb/XcodeGen/pull/1130#discussion_r715083051 
What i want to say is here, sorry for the few context in my comments. I already asked @mthole to do it in other PR.